packages/nimble-components/src/anchor-button/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { css } from '@microsoft/fast-element';
+import {
+    primaryButtonStyles,
+    styles as buttonStyles
+} from '../patterns/button/styles';
+import { buttonLabelDisabledFontColor } from '../theme-provider/design-tokens';
+
+export const styles = css`
+    ${buttonStyles}
+    ${primaryButtonStyles}
+
+    .control {
+        text-decoration: none;
+    }
+
+    :host(:not([href])) {
+        color: ${buttonLabelDisabledFontColor};
+        cursor: default;
+    }
+
+    :host([disabled]) .control {
+        pointer-events: none;
+        cursor: default;
+    }
```

Comment:
Looks like all this can go away in the anchor button's CSS, as the shared button CSS does everything we need once we change the selectors to `:host([disabled]) .control` instead of `.control[disabled]`.